src/ray/core_worker/core_worker.h

```diff
@@ -1065,8 +1065,15 @@ class CoreWorker : public rpc::CoreWorkerServiceHandler {
   // Queue of tasks to resubmit when the specified time passes.
   std::deque<std::pair<int64_t, TaskSpecification>> to_resubmit_ GUARDED_BY(mutex_);
 
-  // Plasma Callback
-  PlasmaSubscriptionCallback plasma_done_callback_;
+  // Guard for Plasma Callback Map
+  mutable absl::Mutex plasma_mutex_;
+
+  // Plasma Callbacks
+  absl::flat_hash_map<ObjectID, std::vector<std::function<void(void)>>>
+      async_plasma_callbacks_ GUARDED_BY(plasma_mutex_);
+
```

Comment:
document this?

Reply:
misplaced, this comment is about documenting PlasmaCallback method

Reply:
I'll modify the comment to be more explanative.